blog/2025-12-16-gravitino-1-1-0-release-notes.mdx

```diff
@@ -0,0 +1,136 @@
+---
+title: Apache Gravitino 1.1.0 - An AI-native data metadata platform
+slug: gravitino-1-1-0-release-notes
+authors: [yuqi1129]
+tags: [apache,gravitino,metadata,multicloud,model,security,government]
+---
+
+
+We are glad to announce the release of Gravitino 1.1.0! This release builds upon the solid foundation laid by Gravitino 1.0.0, introducing a range of new features, improvements, and bug fixes that enhance the platform's capabilities, performance, and security.
+
+
+## Highlights
+- Broader catalog support (initial Lance REST service, a reusable lakehouse-generic catalog, and Hive3) to simplify integration with diverse lakehouse deployments.
+- Multi-cluster fileset support and Python client improvements for real-world multi-region and migration workflows.
+- Stronger metadata-level authorization and security hardening for the Iceberg REST surface.
+- Stability, performance and observability work across the entity-store, caches, scan planning, connectors and CI — reducing operational friction and test flakiness.
+
+
+## New Features
+
+
+1. **Built for the Future of AI Data: Lance REST service** (https://github.com/apache/gravitino/issues/8889).
```

Comment:
A standard link format is `[xxx](http://xxx)`. Does your style really work?

Reply:
All have changed.